mlops/parallelm/mlops/mlops_metrics.py

```diff
@@ -37,13 +37,32 @@ class MLOpsMetrics(object):
 
     >>> mlops.metrics.explained_variance_score(y_true=labels_actual, y_pred=labels_pred)
 
+    >>> mlops.metrics.mean_absolute_error(y_true=labels, y_pred=labels_pred)
+
```

Comment:
please remove empty lines

Reply:
added line to distinguish different ways of using mlops metrics apis. :)